src/content/whats-new/2023/05/whats-new-05-05-dive-into-nrql-from-logs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Unleash your log data analysis with NRQL'
+summary: 'Do you want to unlock deeper insights from your log data in New Relic? We've got you covered! Now you can dive into the NRDB Query Builder directly from your logs without losing context.'
+releaseDate: '2023-05-05'
+---
+You know the power of New Relic when it comes to making sense of the logs generated by your systems. But sometimes, you need to filter, facet, and aggregate your log data in a specific way to get the insights you need.
+
+The NRQL query builder provides a rich set of functionality to help you filter and aggregate log data in a precise and efficient manner with autocompletion, syntax highlighting, and query validation to make it easy to get the answers you need.
```

Comment:
**Q**uery **B**uilder.

We should also mention other features: "Save time by starting a search with one of your most recent queries. Make complex searches readable with multi-line queries. And choose from any of New Relic's chart visualizations to tell a clear story with your log data."